presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -164,6 +164,17 @@ public void testTableWithNoSupportedColumns()
         }
     }
 
+    @Test
+    public void testInsertWithNotSupportedColumn()
```

Comment:
`testInsertInPresenceOfNotSupportedColumn`?

also, add a similar test case for mysql, sql server too